clients/feeder/feeder.go

```diff
@@ -180,10 +159,8 @@ func NewClient(clientURL string) *Client {
 	return &Client{
 		url:        clientURL,
 		client:     http.DefaultClient,
-		backoff:    ExponentialBackoff,
-		maxRetries: 10, // ~40 secs with default backoff and maxWait (block time on mainnet is 20 seconds on average)
-		maxWait:    4 * time.Second,
-		minWait:    time.Second,
+		backoff:    utils.NewNopBackoff(),
```

Comment:
Yes that's the new default. I kind of changed it on a whim while doing the refactor. Would you prefer leaving the previous default?

Reply:
Yeah, `sync` relies on this backoff to exist. Otherwise we will bombard the gateway with requests

Reply:
Will change it in a bit. I modified `node.go` to include backoff, though, so `sync` should still backoff in the current implementation

Reply:
(just realized that creating a new backoff instead of letting the user pass it in has some unwanted side effects; the current implementation works, so I'm sticking with it for now)